### PR TITLE
help: Update documentation concering raw types

### PR DESCRIPTION
**Pull request by brianlheim**
Fix for #2662.

### CODE REVIEW

**Comment by vivid-synth** [comment]
Not sure where you want to put it but maybe you can kill 2 birds with one stone?: https://github.com/supercollider/supercollider/issues/1874

**Comment by brianlheim** [comment]
Updated, this now includes a minor rewrite of the Literals reference file to better reflect the operation of the language, as well as modifications in other helpfiles to fix #1874 and #2667.

File: HelpSource/Classes/Char.schelp [hunk 1]
```diff
@@ -4,25 +4,36 @@ categories:: Core
...
-Returns code::($ )::
+Single space code::($ )::.
+
 method::comma
-Returns code::($\,)::
+Comma code::($,)::.
+
 method::bullet
-Returns a bullet character (•) in SuperCollider.app on macOS, but a simple asterisk (*) on other frontends.
-This method is not recommended, since it's actually not cross-platform.
```
Comment (telephon) [inline HelpSource/Classes/Char.schelp]: long long ago i had added that. we should deprecate it, i think

Comment (brianlheim) [inline HelpSource/Classes/Char.schelp]: OK! After this PR clears I'll make a new one to deprecate it. I want to keep this PR focused on updating documentation. Thanks for the suggestion, I was wondering about this while I was working on it. :)

**Comment by brianlheim** [comment]
At @snappizz 's request, I also added documentation to help explain how character encodings work in SC (and in the SC-IDE).

**Pull request merged**